src/main/java/org/betonquest/betonquest/modules/logger/PlayerLogWatcher.java

```diff
@@ -0,0 +1,110 @@
+package org.betonquest.betonquest.modules.logger;
+
+import org.betonquest.betonquest.modules.logger.handler.chat.PlayerPackageReceiverSelector;
+import org.betonquest.betonquest.modules.logger.handler.chat.ReceiverSelectorRegistry;
+import org.betonquest.betonquest.modules.logger.handler.chat.RecordReceiverSelector;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.logging.Level;
+
+/**
+ * Facade to control what players are seeing which log records that are being logged.
+ */
+public class PlayerLogWatcher {
```

Comment:
I am not sure why i am so confused by this class. Maybe because before the general LogWatcher exists and this is not a specific PlayerLogWatcher. Could it be possible, that this should may be be in an other package in custom? Or something else is strange, that i don't see.